components/ModEditSubmissionForm.vue

```diff
@@ -803,7 +803,7 @@ async function submitUpdatedSubmission(e: Event) {
     }
     try {
         const result = await moderationSubmissionStore.updateSubmission(submissionInputVariables)
-        const submissionResult = result as { updateSubmission: Submission }
+        const submissionResult = result as unknown as { updateSubmission: Submission }
```

Comment:
@ermish It says because of the generic data type within the function call here I have to convert it to unknown since it uses `T` in the utility function before converting it to its type

Reply:
re-reviewed, for this, look at where we define `T`. You can set constraints for T